api/ModelHandler.py

```diff
@@ -0,0 +1,314 @@
+import os
+import PeakError
+import threading
+import api.plLocks as locks
+import pandas as pd
+import api.TrackHandler as th
+import api.PLConfig as pl
+import api.JobHandler as jh
+
+summaryColumns = ['regions', 'fp', 'possible_fp', 'fn', 'possible_fn', 'errors', 'penalty', 'numPeaks']
+modelColumns = ['chrom', 'chromStart', 'chromEnd', 'annotation', 'height']
+
+
+def getModel(data):
+    data['hub'] = data['name'].split('/')[0]
+
+    data['track'] = data['name'].split('/')[-1]
+
```

Comment:
save data['name'].split('/') as intermediate variable since you do this twice

Reply:
This would make more sense. I was doing it this way at some point but I'm not sure why I stopped..